DEV-3/DEV-3/NumeralSystemConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_3
+{
+    /// <summary> 
+    /// This class converts decimal number in other numeral system
+    /// </summary>
+    class NumeralSystemConverter
+    {
+        /// <summary> 
+        /// Method DecimalToOtherNumeralSystem
+        /// A method that creates a string that is a representation of a number in the new numeral system
+        /// </summary> 
+        /// <returns>Value In New System</returns>
+        public string DecimalToOtherNumeralSystem(int number, int newBase)
+        {            
+            StringBuilder valueInNewSystem = new StringBuilder();            
+            do
+            {
+                int numberToAdd;
+                numberToAdd = number % newBase;
+                valueInNewSystem.Append(FigureConverter(numberToAdd));
+                number = number / newBase;
+            } while (0 != number);
+            
+            return ReverseString(valueInNewSystem.ToString());
+        }
+
+        /// <summary>
+        /// Converts decimal numeral to new numeral  numeral system
+        /// </summary>
+        /// <param name="numbertForTranslation"></param>
+        /// <returns></returns>
+        private string FigureConverter(int numbertForTranslation)
+        {
+            char figureInNewSystem;
```

Comment:
possibleValues has to be a const class member not to initialize string each time